runtests.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+# pass in --cluster as an argument to start a cluster instead of a single node
+set -e
+trap 'jobs -p | xargs kill -9' EXIT
+
+CLUSTER_URL=http://127.0.0.1:9200
+ES_PATH=elasticsearch
+
+if [ ${TRAVIS} ]; then
+  ES_PATH=./elasticsearch-1.7.3/bin/elasticsearch
```

Comment:
We kind of have to do this because you can't start a cluster with the version of elasticsearch that is shipped with travis.